src/stateChangeCapture.ts

```diff
@@ -8,29 +8,38 @@ export interface LastItemState {
 }
 
 export function sanitizePipelineName(pipelineName: string | undefined): string {
-
   const sanitisePatterns = process.env.SANITISE_PATTERNS || "-pipeline"
   const removeWhitespace = (x: string) => x.trim()
 
   if (pipelineName) {
     const sanitise = pattern => {
-      const matchResult = pipelineName.split(pattern)
-      if (matchResult.length > 1) {
-        return matchResult[0]
-      } else {
-        //no match
-        return undefined
+      if (pattern.indexOf('*') !== -1) {
+        const StringBefore = pattern.split('*')[0]
```

Comment:
I think i need to reflect on this a little more - check the implications with the info we are storing...will check back later

Reply:
Ok, as a general rule, I think supporting wildcards, etc should be fine.  
We store the pipline names in Dynamo, but query based on the pipeline name itself, rather than the sanitised pattern.  So infix/postfix querying shouldn't be an issue.